robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -649,23 +653,51 @@ function sendRequest (device, command, callback, timeout = REQUEST_TIMEOUT) {
   })
 }
 
+/*
+returns the IP portion of the currently set ROS_MASTER_URI
+*/
+function getRoverIP (callback) {
+  console.log('roverIP: ' + env.ROS_MASTER_IP)
```

Comment:
Use roslogging functions instead of console.log, I'd recommend `logInfo` for these.

Reply:
Fixed